pkg/dashboard/server/flux.go

```diff
@@ -101,3 +114,44 @@ func parseRev(rev string) (string, error) {
 
 	return parts[1], nil
 }
+
+func updateGitopsCommitStatuses(
+	gitopsRepoCache *nativeGit.RepoCache,
+	perf *prometheus.HistogramVec,
+	store *store.Store,
+	repoName, env, eventMessage string,
+	repoPerEnv bool,
+) error {
+	since := time.Now().Add(-24 * time.Hour)
```

Comment:
Too easy. Why 24hrs, why not 10, or 72?

Organize the code in a way that we don't rely on magic numbers.